htdocs/AjaxHelper.php

```diff
@@ -101,8 +101,9 @@
 }
 
 // Also check the module directory for PHP files
+// remove query string from url after '?' like: ajax/test.php?name=test
+$File     = explode("?", $File)[0];
```

Comment:
Where is this coming up? File is defined as `$File   = $_GET['script'];`, on line 57, so the variable shouldn't have included it in the first place.

Reply:
remove query string from url after '?' like: ajax/test.php?name=test
$File = "ajax/test.php?name=test", it will not be found.
$File = "ajax/test.php" it can be found. (on line 106 )